### DISCUSSION TITLE
The Hubble parameter is not a constant

### DISCUSSION THREAD

**Discussion by VitalyPetrikov** [a-cosmology-group/acg#115] [Cosmology Models]
Let's pre-designate the main parameters (properties) of the subject under consideration.
The Hubble parameter cannot be a constant for the following reasons:
The Hubble parameter is drawn on speed-distance schemes as an averaging of too much spread from the average value, and therefore, even with time (already 100 years), it does not come to a certain undisputed value. At average radii, the Hubble parameter is somehow determined, and with an increase in the speeds of the observed galaxies, all sorts of tricks from the universe are used to preserve it.
The average distances between galaxies are approximately the same throughout cosmos in any direction and are determined by the procedure for the formation of galaxies (see sections 2 and 6). That is, the increase in distances with each farther galaxy is practically linear. Of course, there are regions in cosmos with a thicker existence of galaxies, as, accordingly, there are regions with very large voids, but these facts only confirm the rule. But the velocities of galaxies, when approaching light speed, change more and more to a lesser extent.
And in this regard, the Hubble parameter is curved. It is clear from Figure 3 that if the counting of distances from the center (on the scale of distances) at radii of about 1/3 of the radius of modern cosmos is linear, then the counting of the speed of galaxies on the scale of speeds is heated in place.
Hubble parameter is curved in principle and its curvature is approximately determined by the simplified formula given in the essay (section 4 and Fig. 3). At the same time, all sorts of tricks inspired by all sorts of cunning, followers of the theories of the Big Bang, become useless.
In general, it makes no sense to attach great importance to the Hubble parameter, since it practically does not affect anything and nothing follows from his knowledge. The Hubble parameter is a simple statement of fact. Much more interesting is the question of the area of location of galaxies having near-light speed. 
[VVPetrikov About recession of galaxies 2nd edition.pdf](https://github.com/a-cosmology-group/acg/files/12367884/VVPetrikov.About.recession.of.galaxies.2nd.edition.pdf)

**Comment by RedshiftDrift** [comment]
Nice to see you with us Виталий!

**Comment by OlliSant** [comment]
Yes, it is logically impossible, otherwise it is not the space of the whole universe.

**Comment by khuramonline** [comment]
"speed increases and mass increases" is not a fact. Even official physics has abandoned this faulty concept.

**Reply by Pierre314159** [reply]
The problem here is that mass cannot be measured by any other ways than by how it accelerates when subjected to a force. As a massive object’s velocity approaches the speed of light, it takes more and more energy to accelerate it further, and this is observed routinely in particle accelerators. This is a requirement of special relativity, otherwise, massive objects could be accelerated easily further than the speed of light.

**Reply by ftessier** [reply]
Indeed modern physics abandoned the concept of a "relativistic mass" early on (starting with Einstein himself!), and we only speak of the rest mass and total energy. Although colloquially we might still evoke a larger mass, because, as @Pierre314159 says, we can interpret _some_ measurements as if the mass increases. So it is not "faulty", merely a picture that is not very useful and overly restrictive.

**Comment by HanDeBruijn** [comment]
Quotes from the [reference](https://github.com/a-cosmology-group/acg/files/12311102/VVPetrikov.About.recession.of.galaxies.2nd.edition.pdf):

_If the universe consists of space and a radially moving hot shell around it, then it should be considered natural that the microwave background radiation comes out continuously_ $\mbox{ }^{19}$.
$\mbox{ }^{19}$ [_Mathematical formalization of the model_](http://www.alternatievewiskunde.nl/cosmology/CMBR.htm) [?] _isn’t the aim of this work_.
$\mbox{ }^{22}$ _The author's task did **not** include physical and mathematical formalization of the considered model._ 

[D&eacute;j&agrave; vu](https://thisislanduniverse.com/2022/08/24/the-radiation-density-gradient/). Who's task would it be it to **include** physical and mathematical formalization?

**Comment by VitalyPetrikov** [comment]
HanDeBruijn has written:
 Who's task would it be it to include physical and mathematical formalization?

Let those who are not lazy do it

**Comment by VitalyPetrikov** [comment]
Pierre and Fessier, you are both right. It was my school knowledge about the special theory of relativity that manifested itself. In fact, the mass does not grow. You will have to make changes to the test essay. Thank you.

**Comment by VitalyPetrikov** [comment]
Translation error - not you, but I have to ....

**Comment by VitalyPetrikov** [comment]
HanDeBruijn,
The fact is that there is enough work here for several mathematicians for several years

**Comment by mikehelland** [comment]
Hubble's parameter is:

$$H = H_0 [\Omega_m (1+z)^3 + \Omega_\Lambda + \Omega_r (1+z)^4]^{1/2}$$

This is what $H / H_0$ looks like.

![](https://raw.githubusercontent.com/mikehelland/hubbles-law/master/nonexpanding/ep4/expansionrate.png)

It's interesting to note that while it is commonly said that gravity (so the $\Omega_m$ part) pulls back against the expansion rate.

You would think that would mean when the universe is denser, the expansion rate was less. But Hubble's parameter H represents the expansion rate prior to the effects of gravity being factored in, which has to be enormous to counter act gravity in a much denser universe. 

So it seems it would be more accurate to say more matter and more gravity means a higher expansion rate.

**Reply by bgreenman** [reply]
We don't know

**Reply by RedshiftDrift** [reply]
The Big Bang gave the initial kick to expand the early universe at a very high rate, implied by a very large $H$.  But $\Omega_M (1+z)^3$ was also very large, and pulled back against the fast expansion to slow it down to today's $H_0$.
More matter density means more slowing down of the early universe.

>  more matter and more gravity means a higher expansion rate

No, correlation does not mean causation; $\Omega_M$ pulls back against expansion.

**Reply by mikehelland** [reply]
I get that.

Think about what happens in the future in LCDM. The volume increases, the matter density decreases. What happens. The expansion rate goes down.

**Comment by bgreenman** [comment]
We don't know the level of matter needed to either slow or speed up so-called expansion, if expansion really occurs
